docs/user_guides/tls_proxy.md

```diff
@@ -419,16 +416,61 @@ openssl x509 -req -days 3650 -in client.csr -out client.pem -CA ca.pem -CAkey ca
 ```
 </details>
 
+The ```openssl.cnf``` file is a default configuration file of the openssl. If you want to know more details, visit [the manual page](https://www.openssl.org/docs/manmaster/man5/config.html). The ```gen.sh``` file can generate some related certificate files. And you can modify this file to customize for specific demands, such as the valid period of the certificate file, the length of the certificate key or the certificate file names.
+
+In the ```gen.sh``` file, these variables are very important, because they are required in the process of creating a certificate signing request file. The first five variables are the basic signing information, including country, state, location, organization, organizational. CommonName is a useful configuration that deserves your attention. Because this configuration will be verified when the client and the server communicate with each other
+
 ### 3. Run gen.sh to create certificate.
 ```shell
 chmod +x gen.sh
 ./gen.sh
 ```
-They will creat 9 files about ca.key, ca.pem, ca.srl, server.key, server.pem, server.csr, client.key, client.pem, client.csr.
```

Comment:
The following nine files will be created: `ca.key`, `ca.pem`, `ca.srl`, `server.key`, `server.pem`, `server.csr`, `client.key`, `client.pem`, `client.csr`.